eisbuk-admin/src/AppContent.tsx

```diff
@@ -50,12 +51,14 @@ const AppContent: React.FC = () => {
       <PrivateRoute exact path={PrivateRoutes.Root} component={DashboardPage} />
       <PrivateRoute path={PrivateRoutes.Atleti} component={AthletesPage} />
       <PrivateRoute path={PrivateRoutes.Prenotazioni} component={SlotsPage} />
+
       <Route
         path={`${Routes.CustomerArea}/:secretKey/:customerRoute?`}
         component={CustomerAreaPage}
       />
       <Route path={Routes.Unauthorized} component={Unauthorized} exact />
       <Route path={Routes.Debug} children={<DebugPage />} />
+      <Route path={Routes.AttendancePrintable} component={AttendanceSheet} />
```

Comment:
Good job, only use the different component (as mentioned above), also, this route should only be accessible to admin, so it also makes sense to put it above with the other private routes